kpops/components/base_components/base_defaults_component.py

```diff
@@ -238,8 +238,9 @@ def defaults_from_yaml(path: Path, key: str) -> dict:
     value = content.get(key)
     if value is None:
         return {}
+    default_path = path.relative_to(Path.cwd())
     log.debug(
-        f"\tFound defaults for component type {typer.style(key, bold=True, fg=typer.colors.MAGENTA)} in file:  {path}"
+        f"Found defaults for component type {typer.style(key, bold=True, fg=typer.colors.MAGENTA)} in {default_path}"
```

Comment:
why did you decide against the indent?

Reply:
I just seemed weird to have the indent. We have no indents anywhere that is why I have it removed